src/exercises/day-4/async.md

```diff
@@ -0,0 +1,3 @@
+# Exercises
+
+TBD
```

Comment:
I think you actually decided on this in #557?

Reply:
I did, and was planning to land this PR before that one was done. I suppose I can just put it in here now :)